lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -406,6 +406,30 @@ protected function transformReportbackItem($data) {
   }
 
   /**
+   * Transform Signup data and prepare for API response.
+   *
+   * @param  object  $data Signup object.
+   *   An object containing properties of Signup data:
+   *   - id: (string) The Signup id.
+   *   - created_at: (string) Date Signup was created.
+   *   - campaign: (string) Campaign Signup belongs to.
+   *   - campaign_run: (string) Campaign run Signup belongs to.
+   * @return array
+   */
+  protected function transformSignup($data) {
+    return [
+      'signup_id' => $data->id,
+      'source' => $data->source,
+      'created_at' => $data->created_at,
+      'campaign' => [
```

Comment:
got it, makes sense! I wasn't sure about that so commented it out. I'll re-add now.  

Should we also include the `campaign_run` for the signup in `SignupTransformer.php`'s `transform` method? If so, I can just add that function to `Transformer.php` and call that under `$data['campaign`]...` in `SignupTransformer.php`'s `transform` method.